packages/cbioportal-utils/src/ptm/PtmUtils.spec.ts

```diff
@@ -341,4 +352,19 @@ describe('PtmUtils', () => {
             );
         });
     });
+    describe('partial Uniprot data', () => {
+        it('test data with partial of fields', () => {
```

Comment:
Very minor but might be better to have two separate `it`s. One named `uses default ptm type when description is empty`, and the other named `returns no pubmed ids when evidence is empty` or something like that.